src/modules/users/users.service.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Injectable,
+  ConflictException,
+  NotFoundException,
+} from '@nestjs/common';
+import { CreateUserDto, UpdateUserDto } from './dto/user.dto';
+import { DatabaseService } from '../database/database.service';
+
+@Injectable()
+export class UsersService {
+  constructor(private database: DatabaseService) {}
+  async create(createUserDto: CreateUserDto) {
+    try {
+      const newUser = await this.database.createUser(createUserDto);
+      return {
+        ...newUser,
+        id: newUser.id.toString(),
+        createdAt: newUser.createdAt.toISOString(),
+        updatedAt: newUser.updatedAt.toISOString(),
```

Comment:
I'm not sure about this three values, I think ...newUser is enough to return, also if you want to return a string it is better to transform it into string using JSON.stringify(valueToBeString)